### DISCUSSION TITLE
Questions about the evaluation

### DISCUSSION THREAD

**Discussion by ForestsKing** [PaddlePaddle/PaddleSpatial#42] [BaiduKDDCup2022] [answered]
Assuming that 15 days of data are predicted and the final output is (134,1856,288,1), then the final score is calculated as follows:
1. Calculate MAE and RMSE for each turbine for each moment (288, 1)
2. Average the MAE and RMSE for the 1856 moments
3. Sum the MAE and RMSE for the 134 turbines
Am I understanding this correctly?

**Comment by zhoujingbo** [comment, marked as answer]
Hi,

We have updated the method to calculate the evaluation score.  

Please refer to our code as the golden rule for computing the final score : https://github.com/PaddlePaddle/PaddleSpatial/tree/main/apps/wpf_baseline_gru

Thanks a lot for your support

Regards
